### ISSUE TITLE
Question in StepLR

### ISSUE THREAD

**Issue by Fasmanuel2002** [pytorch/pytorch#182628]
There is the following issue on this page: https://docs.pytorch.org/docs/2.11/generated/torch.optim.lr_scheduler.StepLR.html

I have a question: Can you choose the number of steps you want to perform in machine learning in PyTorch, or do the steps depend on the batch size and the length of the examples? Also, with StepLR, can you choose when to change the learning rate between these steps?
